PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -592,6 +587,10 @@ public string GetEffectsLabButtonLabel(Office.IRibbonControl control)
         {
             return TextCollection.EffectsLabButtonLabel;
         }
+        public string GetSyncButtonLabel(Office.IRibbonControl control)
+        {
+            return TextCollection.SyncLabButtonLabel;

```

Comment:
use action framework for this as well.
codes here should be generic and to be used for all features